### ISSUE TITLE
`prefer-event-target`: false positive for Angular application

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#2155]
```ts
import { EventEmitter } from '@angular/core'

export class Foo {
  @Output()
  change = new EventEmitter()
}
```

**Comment by JounQin** [comment]
duplicate of #1853
